frontend/src/lib/components/Orgs/OrgMemberTable.svelte

```diff
@@ -0,0 +1,148 @@
+<script lang="ts">
+  import { Button } from '$lib/forms';
+  import t from '$lib/i18n';
+  import { Icon } from '$lib/icons';
+  import { createEventDispatcher } from 'svelte';
+  import Dropdown from '../Dropdown.svelte';
+  import type { OrgRole, User } from '$lib/gql/types';
+  import OrgRoleText from './OrgRoleText.svelte';
+  import IconButton from '../IconButton.svelte';
+  import AddOrgMemberModal from './AddOrgMemberModal.svelte';
+  import OrgRoleSelect from '$lib/forms/OrgRoleSelect.svelte';
+
+  type TableUser = Pick<User, 'name' | 'locked' | 'username' | 'email' | 'emailVerified'>;
+
+  type Member = {
+    id: string
+    user: TableUser
+    role: OrgRole
+  };
+
+  export let orgId: string;
+
+  export let shownUsers: Member[];
+  export let canManage = false;
+
+  let editingUser: TableUser|undefined = undefined;
+
+  const dispatch = createEventDispatcher();
+
+  let addOrgMemberModal: AddOrgMemberModal;
+  async function openAddOrgMemberModal(): Promise<void> {
+    await addOrgMemberModal.openModal();
+  }
+</script>
+
+<div class="overflow-x-auto @container scroll-shadow">
+  <table class="table table-lg">
+    <thead>
+      <tr class="bg-base-200">
+        <th>
+          {$t('admin_dashboard.column_name')}
+          <span class="i-mdi-sort-ascending text-xl align-[-5px] ml-2" />
+        </th>
+        <th>{$t('admin_dashboard.column_email_or_login')}</th>
+        <th class="@2xl:table-cell">
+          {$t('admin_dashboard.column_role')}
+        </th>
+        <th class="px-2 py-1">
+          {#if canManage}
+            <Dropdown>
+              <IconButton icon="i-mdi-plus" variant="btn-success" size="btn-sm" outline={false} />
+              <ul slot="content" class="menu">
+                <li>
+                  <button class="whitespace-nowrap text-success" on:click={openAddOrgMemberModal}>
+                    <Icon icon="i-mdi-plus" />
+                    {$t('org_page.add_user.add_button')}
+                  </button>
+                </li>
+                <!-- TODO: Do we want this on the org page? -->
+                <!-- <li>
+                  <button class="whitespace-nowrap" on:click={() => dispatch('filterProjectsByUser', user)}>
+                    <Icon icon="i-mdi-filter-outline" />
+                    {$t('project.filter.filter_user_projects')}
+                  </button>
+                </li> -->
+              </ul>
+            </Dropdown>
+          {/if}
+        </th>
+      </tr>
+    </thead>
+    <tbody>
+      {#each shownUsers as member}
+      {@const user = member.user}
+        <tr>
+          <td>
+            <div class="flex items-center gap-2 max-w-40 @xl:max-w-52">
+              <Button variant="btn-ghost" size="btn-sm" class="max-w-full" on:click={() => dispatch('openUserModal', user)}>
```

Comment:
This is not implemented yet

Reply:
Yep. Working on it alongside the editUser modal that the select-in-the-table is going to turn into. Won't get it finished today, but wanted to acknowledge your comment so you know I've seen it and will work on it.

Reply:
Don't forget: we already have a modal for displaying the user's details! 🙂

Reply:
I didn't forget; as you'll see in commit 9f2e0d6c, I used the same modal.